commands/gist.go

```diff
@@ -141,18 +146,17 @@ func createGist(cmd *Command, args *Args) {
 
 func showGist(cmd *Command, args *Args) {
 	args.NoForward()
-	if args.ParamsSize() < 1 {
-		utils.Check(cmd.UsageError("you must specify a gist ID"))
```

Comment:
How come you removed this? Now the `hub gist show` command (no arguments) will crash instead of gracefully error out with the above message.

Reply:
OHHH because I misread - I thought this was checking for a filename. Sorry I'll revert that.